app/View/LayoutForm.cs

```diff
@@ -53,15 +61,36 @@ private void custumizeSideForm()
         // Metodo que cambia el panel principal de la derecha
         public void changeLayout(Form newForm)
         {
-            if (activeLayoutForm != null)
-                activeLayoutForm.Close();
-
+            if (activeLayoutForm != null) activeLayoutForm.Close();
+            
             activeLayoutForm = newForm;
-            custumizeLayoutForm();
+
+            if (!authenticate) resizeWindowToLogin();
+
+            customizeLayoutForm();
+        }
+        // Método que redimensiona layoutForm unicamente en la ventana de login
+        public void resizeWindowToLogin()
+        {
+            panelSide.Visible = false;
+            this.Width = loginFormWidth;
+            this.MaximizeBox = false;
+            this.FormBorderStyle = FormBorderStyle.FixedSingle;
+        }
+
+        // Método que redimensiona layoutForm a su tamaño normal despúes del login
+        public void resizeWindowsToNormal()
+        {
+            authenticate = true;
+            panelSide.Visible = true;
+            this.MaximizeBox = true;
```

Comment:
porque this.MaximizeBox es necesario activarlo, creo que por defecto todo lo pusimos en false, supongo que es mejor mantener todo sin permitir eso, para mantener la ventana estatica en sus tamaños